process-automation/latest/model/connectors/docusign.md

```diff
@@ -2,103 +2,214 @@
+
+| Parameter | Type | Description |
+|--- | --- | --- |
+| envelopeId | String | Envelope ID of the document. |
+| status | String | Status of the envelope. |
+| URI | String | URI related to the envelope. |
+| docusignOutput | JSON | DocuSign output after sending the document for signature. |
 
-An example of the `targetFileMetadata` that can be stored with the document is:
+The input parameters for `DOWNLOAD_DOCUMENT` are:
 
-```json
+| Parameter | Type | Description |
+| --------- | ---- | ----------- |
+| outputFileName | String | *Optional.* The name of the file that will be created, for example `invoice.pdf`. |
+| envelopeId | yes | *Required* UUID related to the envelope from the DocuSign API. |
+| targetFolder | Folder | *Requires one.* A [variable]({% link process-automation/latest/model/processes/index.md %}#process-variables) of type folder to store the signed document in. |
+| targetFolderId | String | *Optional.* The target folder ID to save the document in Content Services. |
+| targetFolderPath | String | *Requires one.* The location path or relative path of the folder to store the signed document in Content Services. For example, a location path: `/app:company_home/app:user_homes/cm:hruser` or a relative path: `/User Homes/hruser`.  |
+| targetFileType | String | *Optional.* The type to set for the signed file, for example `fin:invoice`. |
+| targetFileMetadata | content-metadata | *Optional.* Metadata assigned to the signed document in Content Services. |
+| underscoreMetadata | boolean | *Optional.* If set to `true` the received prefixed properties names contain underscores (_) instead of colons (:), for separating the namespace prefix from the property name. |
+
+The output parameters from `DOWNLOAD_DOCUMENT` are:
+
+| Property | Type | Description |
+|--- | --- |
+| file | file | The file stored in Content Services of the document. |
+
+The following is an example of the POST body for the Activiti REST API `http://{{domain}}/{{applicationName}}-rb/v1/process-instances` endpoint:
+
+```JSON
 {
-"ahr:contract-type": "Full Time",
-"ahr:full-name": "John Doe",
-"ahr:role": "Developer"
+  "processDefinitionKey": "DocuSignProcessTest",
+  "processInstanceName": "processDocuSignTest_Simple",
+  "businessKey": "MyBusinessKey",
+  "variables": {
+  	"recipientEmail" : "test@test.com",
+  	"recipientName" : "AAE Test",
+  	"emailSubject" : "Sign this document",
+    "files" : [
+        {
+  	        "nodeId" : "<node-id-of-document-to-be-signed>"
+        }
+    ],
+  	"outputFileName" : "testFileName.docx",
+  	"nodeFormat" : "docx",
+  	"documentId" : "10",
+  	"signHerePage" : "1",
+  	"signHereX" : "100",
+  	"signHereY" : "200",
+  	"timeout: "3600"
+  },
+  "payloadType":"StartProcessPayload"
 }
 ```
 
-The output parameters from signing a document are:
+In the business process definition, the service task called **docuSignTask** has the implementation attribute configured to use the connector.
```

Comment:
Same as above. I think this is more technical. Is there any kind of documentation like this in other sections? I can't see something similar in the rest of the connectors, for example.